custom/icds_reports/management/commands/sql_scripts/update_ccs_data.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS temp_ccs_table;
+CREATE TABLE temp_ccs_table
```

Comment:
why do you need to create a temp table here? moreover I don't think this query would work as below you can't update a distributed table using a non distributed table. I think you don't need a temp table, you can directly update it

Reply:
I added a step after this where it's converted to distributed and the point of using temp table is to make query little simplified.

Reply:
Oh I did not see that part.